### ISSUE TITLE
Adenosine triphosphate (ATP) low in erythrocytes

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9753]
**Preferred term label:**
Adenosine triphosphate (ATP) low in erythrocytes

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Hemolytic anemia due to elevated adenosine deaminase https://omim.org/entry/301083
Also see https://www.ncbi.nlm.nih.gov/medgen/C1863224

We already have a pending new term request for "Adenosine triphosphate (ATP) high in erythrocytes" (elthat_130818185943).  This is the opposite and it may be useful to do both together..

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
added as HP:4000184  Reduced erythrocyte adenosine triphosphate concentration
